README.md

```diff
@@ -1,3 +1,18 @@
 # wtf-explore-libraries
 
-Activity prompt for "What They Forgot To Teach You About R" workshop
+
+Get started via
+
+```r
+usethis::use_course("rstats-wtf/wtf-explore-libraries")
+```
+
+Choose your own adventure!
+
+* `01_explore-libraries_spartan.R` (directions to explore without suggested code)
+
+* `01_explore-libraries_comfy.R` (directions to explore with suggested code)
+
+* `01_explore-libraries_jenny.R` (directions to explore with code solutions)
```

Comment:
I wonder if we should replace "jenny" with "solution"? No strong opinion, but I offer it for your consideration.

Reply:
that makes sense, then agnostic to whoever teaches